_maps/map_files/generic/CentCom_skyrat_z2.dmm

```diff
@@ -6549,6 +6549,11 @@
 /obj/effect/turf_decal/vg_decals/numbers/six,
 /turf/open/floor/iron/dark,
 /area/cruiser_dock)
+"dWZ" = (
+/turf/open/misc/beach/coastline_b{
```

Comment:
Is there no directional variants that already exist in the code, so you don't have to set the dir yourself? If not, it's fine, I suppose, but still could be nice.

Reply:
No directional variants, had to do it myself